Concentus.Oggfile/OpusHeader.cs

```diff
@@ -11,10 +11,35 @@ internal class OpusHeader
         byte version;
         byte channel_count;
         ushort pre_skip;
-        uint input_sample_rate;
+        internal uint input_sample_rate;
         short output_gain;
         byte mapping_family;
         byte stream_count;
         byte coupled_count;
+    
+        internal static OpusHeader ParsePacket(byte[] packet, int packetLength)
+        {
+            if (packetLength < 19)
+                return null;
+
+            if (!"OpusHead".Equals(Encoding.UTF8.GetString(packet, 0, 8)))
```

Comment:
this check is already made before the method is called

Reply:
Yes, it's redundant a little bit. 
I've used the same style as originally has been used for OpusTags.ParsePacket(), so I'd keep it.